### ISSUE TITLE
ubus call: Command failed: Unknown error

### ISSUE THREAD

**Issue by systemcrash** [openwrt/luci#7321]
ping @jow- 

I'm stumped on this one.  Please install luci-app-lldpd ( on 23.05.x ) and once it's running, try to view the status tab. It runs:

```sh
ubus call luci.lldpd getStatus
```

from `/usr/share/rpcd/ucode/luci.lldpd`

It currently returns: `Command failed: Unknown error`. `strace` is not much help either.

But running the individual commands contained in the ucode file work fine:

`# lldpcli -f json0 show statistics`
```json
{
  "lldp": [
    {
      "interface": [
        {
...
```

I checked for BOM or other weird UTF-8 stuff. No dice. 

This has worked previously, so I suspect some update to rpcd ...?

**Comment by systemcrash** [comment]
I think I found the problem. Seems to be in the result of `popen()`.

**Comment by systemcrash** [comment]
Closed by c40253ada3d3e0257e1e3796aeab5ad46bc02f5b
